src/dxvk/dxvk_device.h

```diff
@@ -472,6 +472,8 @@ namespace dxvk {
     DxvkStatCounters            m_statCounters;
     
     DxvkDeviceQueueSet          m_queues;
+
+    std::atomic<uint32_t>       m_numSamplers = { 0 };
```

Comment:
I still don't like this at all, and would like to ask to drop the sampler HUD thing for now.

Instead of tracking sampler count in the backend (which is *completely* pointless for D3D11 and just confusing, and also incredibly ugly code-wise), we should probably work on making the HUD a bit more flexible to allow this to be done on the D3D9 side.

Reply:
Alright, I've dropped the commit